bat/tests/customize-os-release/run.bats

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bats
+
+# shared test functions
+load ../../lib/mixerlib
+
+setup() {
+  global_setup
+}
+
+@test "Customize os-release file" {
+
+  mixer init > /dev/null
+
+  #Backup orig config and use modified config which has customized os-release file
+  mv builder.conf builder-orig.conf
```

Comment:
No need to backup the original config here.

Reply:
I will remove it.